build-coordinator/src/main/java/org/jboss/pnc/core/builder/BuildScheduler.java

```diff
@@ -29,5 +29,6 @@
  * @author <a href="mailto:matejonnet@gmail.com">Matej Lazar</a>
  */
 public interface BuildScheduler {
+    boolean hasBeenBuilt(BuildTask buildTask);
```

Comment:
1. I don't see a reason for a different requirement to check if the builds exists or not. And if they do we should extract that functionality to a new class rather than scheduler. p.s.: local is meant as local to coordinator, which is likely to be dropped once BPM work-flow is in completely in place.

2. the check is already (this PR) called in the BuildCoordinator, which is OK, I would move just the method that does the checking.